### PR TITLE
1895 document select view

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

**Review by codyseibert — APPROVED** [review]

File: web-client/src/ustc-ui/DocumentSelect/DocumentSelect.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+              padding-top-2 ${required && 'padding-bottom-2'}
+              `}
+            >
+              <span>{name} </span>
+              {!required && <span className="usa-hint">(optional)</span>}
+            </div>
+            {documentSelectedForScan !== value && (
+              <div className="grid-col-2 padding-top-2">
+                <button
+                  aria-controls={`add ${name} file`}
```
Comment (kkoskelin) [inline web-client/src/ustc-ui/DocumentSelect/DocumentSelect.jsx:36]: the `aria-controls` attribute should equal an element ID elsewhere in the page.

File: web-client/src/views/ScanBatchPreviewer.jsx [hunk 2]
```diff
@@ -263,15 +264,15 @@ export const ScanBatchPreviewer = connect(
...
           <div
             className={`usa-form-group ${
-              validationErrors.petitionFile ? 'usa-form-group--error' : ''
+              validationErrors[documentType] ? 'usa-form-group--error' : ''
             }`}
           >
             <input
               accept=".pdf"
-              aria-describedby="petition-hint"
+              aria-describedby={`${documentType}-hint`}
```
Comment (kkoskelin) [inline web-client/src/views/ScanBatchPreviewer.jsx:272]: the value of this attribute ought to also correspond to the ID of an element which describes it.

File: web-client/src/views/ScanBatchPreviewer.jsx [hunk 3]
```diff
@@ -311,11 +312,11 @@ export const ScanBatchPreviewer = connect(
         >
           <div className="grid-container padding-x-0">
             <div className="grid-row grid-gap">
-              <div className="grid-col-6">
-                <h3 style={{ marginBottom: '0px' }}>Add {documentType}</h3>
+              <div className="grid-col-8">
+                <h3 style={{ marginBottom: '0px' }}>Add {documentTypeName}</h3>
```
Comment (kkoskelin) [inline web-client/src/views/ScanBatchPreviewer.jsx:316]: `className="margin-bottom-0"` ?

**Review by kkoskelin — COMMENTED** [review]
just a few comments / questions

**Pull request merged**